Cake.Recipe/Content/gitversion.cake

```diff
@@ -24,6 +24,10 @@ public class BuildVersion
 
         if (BuildParameters.ShouldRunGitVersion)
         {
+            if (context.IsRunningOnUnix()) {
```

Comment:
@AdmiringWorm in principle, I have no objection to this getting added, however, I think we might want a way to break out of this happening, if the user doesn't want it.  How about an additional BuildParameter, which is `true` by default added to this if statement.  That way, if can be set to false, if the user doesn't want to do this for some reason.  Thoughts?

Reply:
I would thing that if a user wants to be able to run gitversion on linux, they would have no problems with the change.

The time it takes to run the code as well isn't really noticable, and if the the distro is able to load the included libgit library it won't even load the system library at all.

So IMO, I really see no need to have a BuildParameter to enable/disable this.